QuickshaRe/Domain/QRPictureGenerator.swift

```diff
@@ -13,8 +13,12 @@ final public class QRPictureGenerator {
     
     let context = CIContext()
     
+    private let dummy :Int = 123
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/el-hoshino/QuickshaRe/blob/51d5b333c7816582b25eb2f1e17425454b643af1/QuickshaRe/Domain/QRPictureGenerator.swift#L16"></span>Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.<br />
<code>colon</code> <code>QRPictureGenerator.swift:16</code></td>
    </tr>
  </tbody>
</table>